src/app/onboarding/views/MultiZoneView.vue

```diff
@@ -162,11 +162,11 @@ function clearZoneIngressTimeout() {
 }
 
 .status--is-connected {
-  color: var(--green-500);
+  color: var(--kui-color-text-success);
 }
 
 .status--is-disconnected {
-  color: var(--red-500);
+  color: var(--kui-color-text-danger);
```

Comment:
question: can you use the SCSS tokens here instead?

Reply:
Fixed in https://github.com/kumahq/kuma-gui/pull/1314/commits/b5e11c5c05ecb42b15373dbad1068e27f5796288.